Tests/SpeziLLMTests/LLMOpenAIParameterTests+CustomTypes.swift

```diff
@@ -12,21 +12,34 @@ import XCTest
 
 final class LLMOpenAIParameterCustomTypesTests: XCTestCase {
     struct CustomType: LLMFunctionParameterArrayElement, Encodable, Equatable {
-        static var itemSchema: SpeziLLMOpenAI.LLMFunctionParameterItemSchema = .init(
-            type: .object,
-            properties: [
-                "propertyA": .init(type: .string, description: "First parameter"),
-                "propertyB": .init(type: .integer, description: "Second parameter")
-            ]
-        )
-        
+        static var itemSchema: LLMFunctionParameterItemSchema = {
+            do {
+                return try .init(unvalidatedValue: [
+                    "type": "object",
+                    "properties": [
+                        "propertyA": [
+                            "type": "string",
+                            "description": "First parameter"
+                        ],
+                        "propertyB": [
+                            "type": "integer",
+                            "description": "Second parameter"
+                        ]
+                    ]
+                ])
+            } catch {
+                print("unable to initialse schema in LLMOpenAIParameterCustomTypesTets")
+                return .init()
```

Comment:
What happens down the line if the validation fails and an empty schema is created?

Reply:
Shouldn't something like this do the trick? 

```diff
diff --git a/Tests/SpeziLLMTests/LLMOpenAIParameterTests+CustomTypes.swift b/Tests/SpeziLLMTests/LLMOpenAIParameterTests+CustomTypes.swift
index ca262bd..3bbce43 100644
--- a/Tests/SpeziLLMTests/LLMOpenAIParameterTests+CustomTypes.swift
+++ b/Tests/SpeziLLMTests/LLMOpenAIParameterTests+CustomTypes.swift
@@ -11,6 +11,10 @@ import XCTest


 final class LLMOpenAIParameterCustomTypesTests: XCTestCase {
+    override func setUp() {
+        continueAfterFailure = false
+    }
+
     struct CustomType: LLMFunctionParameterArrayElement, Encodable, Equatable {
         static var itemSchema: LLMFunctionParameterItemSchema = {
             do {
@@ -28,8 +32,7 @@ final class LLMOpenAIParameterCustomTypesTests: XCTestCase {
                     ]
                 ])
             } catch {
-                print("unable to initialse schema in LLMOpenAIParameterCustomTypesTets")
-                return .init()
+                XCTFail("Unable to initialse schema in LLMOpenAIParameterCustomTypesTets")
             }
         }()
```

The compiler is still expecting a `return` after the `XCTFail()`. Do you have another suggestions?